### ISSUE TITLE
Table of Contents Formatting

### ISSUE THREAD

**Issue by webmaster94** [woodworker/womboras-journal-helper#7]
Hello I am having a small issue with this module. The table of contents is a great feature but the formatting appears to be not working correctly. 

I have a journal in the following format:
```
<h1>Foaming Mugs</h1>
<h2>Getting the Quest</h2>
<h3>Treasure</h3>
<h2>Search for the Sled</h2>
<h3>Blizzard</h3>
<h3>Oobok’s Remains</h3>
<h3>Goblins</h3>
<h3>Freeing the Bears</h3>
<h3>Treasure</h3>
<h2>Concluding the Quest</h2>
```

But the table of contents comes out like this:
![image](https://user-images.githubusercontent.com/85364225/149803634-b32cb2da-dd0d-4c63-9290-a90742cb1799.png)

Instead of like this, which I would think would be the way it should be formatted:
![image](https://user-images.githubusercontent.com/85364225/149804579-d057d858-e2fd-4d4e-a95b-81a5c9eebb7c.png)

Any assistance on this would be greatly appreciated.

**Comment by woodworker** [comment]
jeah, i also already stumbled into this issue, i am already looking into this and hope to find a solution for the next update.

**Comment by woodworker** [comment]
This is fixed and released in version 0.0.6

i had a few problems with GIthub Builds and finaly fixed it :D
Next release soon for the new Foundry Versions - as i already updated and need my plugin :D
